test/containers/run-tests

```diff
@@ -55,6 +55,12 @@ def run(opts):
     return testlib.test_main(options=opts, suite=suite)
 
 def main():
+    # HACK: Full chromium under Xvfb is very brittle; disable default parallelism
+    try:
+        del os.environ["TEST_JOBS"]
```

Comment:
Any reason not to set opts.jobs=1 rather than messing with the environment?

Reply:
The idea was to not break the `--jobs` option, just change the default in our containers. I can drop the option here as well, but that seemed more intrusive.

Reply:
Ok, changing the env on the fly always seems a little weird to me. But it's just a temp hack so its fine for now.